src/dune_rules/dune_file.mli

```diff
@@ -473,6 +473,16 @@ module Generate_sites_module : sig
     }
 end
 
+(** Stanza to produce JavaScript targets from Melange libraries *)
+module Melange_stanza : sig
+  type t =
+    { loc : Loc.t
+    ; target : string
+    ; spec : Melange.Spec.t
+    ; libraries : Lib_dep.t list
+    }
+end
```

Comment:
Similarly, this should live in a `Melange_stanzas` module and called after the stanza name (Emit).

Reply:
https://github.com/ocaml/dune/pull/6349/commits/db4924ad38e566225eb68bd9596d2b6d61f2efb7